src/classes/CRLP_RollupCMT_TEST.cls

```diff
@@ -0,0 +1,229 @@
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2019
+* @group Customizable Rollups Operations Services
+* @description Unit Test for the Rollups Metadata
+*/
+
+@isTest(IsParallel=true)
+public with sharing class CRLP_RollupCMT_TEST {
+    
+    /******************************************************************************************************************
+     * @description verifies that similar rollups are not marked as different
+     *****************************************************************************************************************/
+    @isTest
+    private static void similarRollupsAreMarkedAsEqual() {
+
+        CRLP_RollupCMT.Rollup newRollup = generateRollup('1');
+        CRLP_RollupCMT.Rollup existingRollup = generateRollup('2');
+
+        System.assertEquals(false, newRollup.isDifferent(existingRollup), 'Similiar Rollups are marked as as equal');
+    }
+
+    /******************************************************************************************************************
+     * @description verifies that different rollups are marked as different
+     *****************************************************************************************************************/
+    @isTest
+    private static void differentRollupsAreMarkedAsDifferent() {
+
+        CRLP_RollupCMT.Rollup newRollup = generateRollup('1');
+        CRLP_RollupCMT.Rollup existingRollup = generateRollup('2');
+        newRollup.operation = 'Count';
+
+        System.assertEquals(true, newRollup.isDifferent(existingRollup), 'Different Rollups are marked as different');
```

Comment:
For the sake of code coverage of all the if statements, is it worth going through each property (i.e. `filterGroupRecordName`, `operation`, `timeBoundOperationType`, etc.) to make sure a difference in each one would return true?